plenum/test/transactions/test_req_to_txn.py

```diff
@@ -23,25 +24,36 @@ def req_and_expected(request, looper, sdk_wallet_client):
         #     req.pop('signature')
         if request.param == 'no_protocol_vers':
             req.pop('protocolVersion')
+        digest = Request(
+            req.get(f.IDENTIFIER.nm, None),
+            req.get(f.REQ_ID.nm, None),
+            req.get(OPERATION, None),
+            req.get(f.SIG.nm, None),
+            req.get(f.SIGS.nm, None),
+            req.get(f.PROTOCOL_VERSION.nm, None)
+        ).digest
+        sign = req.get("signature")
```

Comment:
please replace "signature" with f.SIG.nm as you use before

Reply:
done